index.html

```diff
@@ -176,6 +175,31 @@ <h5 class="categories-title">Skills</h5>
         </div>
     </div>
 </div>
+<form class="contact_forms" action="index.html" method="post">
+    <p class="text">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+    <ul class="form_field">
+        <li class="form_name">
+            <input type="text" id="first_name" name="first_name" class="input" placeholder="Full Name" minlength="6" required>
```

Comment:
Please remove the `minlength` attribute and add a `maxlength="30"` attribute as requested by the previous reviewer.

Reply:
okay, thank you, let me do it